src/java.base/share/classes/java/lang/reflect/Member.java

```diff
@@ -76,9 +78,24 @@ public interface Member {
      *
      * @return the Java language modifiers for the underlying member
      * @see Modifier
+     * @see #accessFlags()
      */
     public int getModifiers();
 
+
+    /**
+     * {@return an unmodifiable set of the {@linkplain AccessFlag
+     * access flags} for this member, possibly empty}
+     *
+     * @implSpec
+     * The default implementation returns an empty set.
+     * @see #getModifiers()
+     * @since 20
+     */
+    public default Set<AccessFlag> accessFlags() {
+        return Set.of();
```

Comment:
Is is not better to throw a NoSuchMethodError instead of Set.of() if there is no implementation.

Reply:
Actually, it should probably be `UnsupportedOperationException` instead.

Reply:
Hmm. If we had sealed classes and interfaces back in JDK 1.1 when Member was added, it most likely would have been added as sealed interface. But, we didn't have sealed interfaces back then so Member can (potentially) be extended by anyone. IIRC, there are a few classes implementing Member outside of the JDK.

So, when adding a new method would JDK 20, the method should certainly have be default method. I think throwing UnsupportedOperationException in the default is marginally better than the alternatives. I'll update the PR accordingly in a subsequent push. Thanks.